modules/integration_azure-app-service/detectors-app-services.tf

```diff
@@ -1,253 +0,0 @@
-    parameterized_body    = var.message_body == "" ? local.rule_body : var.message_body
-  }
-
-  max_delay = var.heartbeat_max_delay
-}
-
-resource "signalfx_detector" "response_time" {
-  name = format("%s %s", local.detector_name_prefix, "Azure App Service response time")
-
-  authorized_writer_teams = var.authorized_writer_teams
-  teams                   = try(coalescelist(var.teams, var.authorized_writer_teams), null)
-  tags                    = compact(concat(local.common_tags, local.tags, var.extra_tags))
-
-  program_text = <<-EOF
-        base_filter = filter('resource_type', 'Microsoft.Web/sites') and filter('is_Azure_Function', 'false') and filter('primary_aggregation_type', 'true')
-        signal = data('HttpResponseTime', filter=base_filter and ${module.filtering.signalflow})${var.response_time_aggregation_function}.fill(value=None).publish('signal')
-        detect(when(signal > threshold(${var.response_time_threshold_critical}), lasting="${var.response_time_lasting_duration_critical}")).publish('CRIT')
-        detect(when(signal > threshold(${var.response_time_threshold_major}), lasting="${var.response_time_lasting_duration_major}") and (not when(signal > ${var.response_time_threshold_critical}, lasting="${var.response_time_lasting_duration_critical}"))).publish('MAJOR')
-    EOF
-
-  rule {
-    description           = "is too high > ${var.response_time_threshold_critical}s"
-    severity              = "Critical"
-    detect_label          = "CRIT"
-    disabled              = coalesce(var.response_time_disabled_critical, var.response_time_disabled, var.detectors_disabled)
-    notifications         = try(coalescelist(lookup(var.response_time_notifications, "critical", []), var.notifications.critical), null)
-    runbook_url           = try(coalesce(var.response_time_runbook_url, var.runbook_url), "")
-    tip                   = var.response_time_tip
-    parameterized_subject = var.message_subject == "" ? local.rule_subject : var.message_subject
-    parameterized_body    = var.message_body == "" ? local.rule_body : var.message_body
-  }
-
-  rule {
-    description           = "is too high > ${var.response_time_threshold_major}s"
-    severity              = "Major"
-    detect_label          = "MAJOR"
-    disabled              = coalesce(var.response_time_disabled_major, var.response_time_disabled, var.detectors_disabled)
-    notifications         = try(coalescelist(lookup(var.response_time_notifications, "major", []), var.notifications.major), null)
-    runbook_url           = try(coalesce(var.response_time_runbook_url, var.runbook_url), "")
-    tip                   = var.response_time_tip
-    parameterized_subject = var.message_subject == "" ? local.rule_subject : var.message_subject
-    parameterized_body    = var.message_body == "" ? local.rule_body : var.message_body
-  }
-
-  max_delay = var.response_time_max_delay
-}
-
-resource "signalfx_detector" "memory_usage_count" {
```

Comment:
It seems this detector is missing "memory_usage_count"

Reply:
This detector has been deliberately dropped since it was using absolute values.